app/models/non_hmis_client.rb

```diff
@@ -36,13 +36,15 @@ def current_covid_assessment
     where(available: true)
   end
 
-  scope :visible_to, -> (user) do
+  scope :visible_to, ->(user) do
     if user.can_edit_all_clients?
       all
+    elsif user.can_view_all_covid_pathways?
+      where(id: NonHmisAssessment.select(:non_hmis_client_id))
```

Comment:
Should this have a`type` guard, or is it really can view all assessments?

Reply:
So, maybe it should.  The end result is that it is guarded, but in the controller and view.